### DISCUSSION TITLE
React Router v7 Support

### DISCUSSION THREAD

**Discussion by sbeckerdev** [shadcn-ui/ui#5979] [General]
Can somebody help me install it along with React Router v7 ? Darkmode is not working.

**Comment by mdklause** [comment]
I have the same problem, the underlining issue is that the package [remix-themes](https://github.com/abereghici/remix-themes) doesn't support React Router V7 and only supports Remix V2 and V1. I made an [issue](https://github.com/abereghici/remix-themes/issues/47) in the repo, but I sadly don't have the time at the moment to fix this issue myself.

**Reply by sbeckerdev** [reply]
A quick fix is using Next-Themes dark mode, it is working with Remix and React router v7. Sadly, the support from schadcn is very slow, I wish they could fix the issue . I really love the schadcn. I don’t have time to implement a custom darkmode.
